scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,252 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import o3de.command_utils as command_utils
+import o3de.export_project as exp
+import export_utility as eutil
+import o3de.manifest as manifest
+
+from o3de import android, android_support
+
+
+import argparse
+import logging
+import pathlib
+import sys
+
+from typing import List
+
+ASSET_PLATFORM = 'android'
+
+def export_source_android_project(ctx: exp.O3DEScriptExportContext,
+                                  target_android_project_path: pathlib.Path,
+                                  seedlist_paths: List[pathlib.Path],
+                                  seedfile_paths: List[pathlib.Path],
+                                  level_names: List[str],
+                                  asset_pack_mode: str,
+                                  engine_centric: bool,
+                                  should_build_tools: bool,
+                                  should_build_all_assets: bool,
+                                  build_config: str,
+                                  tool_config: str,
+                                  tools_build_path: pathlib.Path | None,
+                                  max_bundle_size: int,
+                                  fail_on_asset_errors: bool,
+                                  deploy_to_device: bool,
+                                  org_name: str|None,
+                                  logger: logging.Logger|None) -> None:
+    o3de_cli_script_path = ctx.engine_path / (f'scripts/o3de{android_support.O3DE_SCRIPT_EXTENSION}')
+
+    if not logger:
+        logger = logging.getLogger()
+        logger.setLevel(logging.ERROR)
+
+    is_installer_sdk = manifest.is_sdk_engine(engine_path=ctx.engine_path)
+
+    android_arg_parser = argparse.ArgumentParser()
+    android_subparser = android_arg_parser.add_subparsers(title="Android sub-commands")
+    android.add_args(android_subparser)
+
+    # run_android_command([o3de_cli_script_path, 'android-configure', '--validate'], cwd=ctx.project_path, error_msg="Invalid Android Configuration")
```

Comment:
Remove commented out code. Suggest that you replace this with a brief description on the approach below instead.